Neos.Neos/Classes/Domain/Service/WorkspaceService.php

```diff
@@ -83,6 +84,12 @@ public function getWorkspaceMetadata(ContentRepositoryId $contentRepositoryId, W
      */
     public function setWorkspaceTitle(ContentRepositoryId $contentRepositoryId, WorkspaceName $workspaceName, WorkspaceTitle $newWorkspaceTitle): void
     {
+        $workspaceMetadata = $this->loadWorkspaceMetadata($contentRepositoryId, $workspaceName);
```

Comment:
This can return `null`. I can fix this tomorrow

Reply:
i think that fetching it here is just needless no? can we not just put the upsert into one query below? And not rely on the affected rows?

I thought it would just be simple to fix :D now im stuck reading through things like this https://sqlperformance.com/2020/09/locking/upsert-anti-pattern 😂

Reply:
Fixed via https://github.com/neos/neos-development-collection/pull/5397